Source/WebKit/UIProcess/Cocoa/WebProcessPoolCocoa.mm

```diff
@@ -1529,6 +1529,8 @@ static void webProcessPoolHighDynamicRangeDidChangeCallback(CFNotificationCenter
             continue;
         RetainPtr cfFontURL = checked_cf_cast<CFURLRef>(CFDictionaryGetValue(fontDictionary.get(), CFSTR("NSCTFontFileURLAttribute")));
         URL fontURL(cfFontURL.get());
+        if (fontURL.string().startsWith("file:///Applications/"_s))
```

Comment:
Apps can live anywhere, not just in /Applications, so this seems wrong.

Also, do we need to allow the process to access these application-specific fonts? Did we in shipping?